Arrowgene.Ddon.Database/Files/Database/Script/schema_sqlite.sql

```diff
@@ -588,7 +588,7 @@ CREATE TABLE IF NOT EXISTS "ddon_system_mail_attachment"
     "attachment_id"   INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
     "message_id"      INTEGER                           NOT NULL,
     "attachment_type" INTEGER                           NOT NULL,
-    "is_received"     INTEGER                           NOT NULL DEFAULT 0,
+    "is_received"     INTEGER                           NOT NULL DEFAULT FALSE,
```

Comment:
shouldnt this data type be changed to BOOLEAN here too

Reply:
yes, fixed